test/actions/source_test.js

```diff
@@ -9,13 +9,13 @@ var config = require('../config');
 var utils = require('../utils');
 
 
-describe('actions/source', function() {
+describe.only('actions/source', function() {
```

Comment:
forgotten only

Reply:
Ah, damn it. We really need a lint step for this…